ci/infra/testrunner/platforms/terraform.py

```diff
@@ -151,6 +151,8 @@ def _update_tfvars(self, tfvars):
             "authorized_keys": [self.utils.authorized_keys()]
         }
 
+        new_vars.update(self.platform_new_vars)
```

Comment:
on on-libvirt platforms this fails because there is no platform_new_vars so we need at least an empty dict as part of this class that other classes can override

Reply:
:man_facepalming: OMG you are right, fixed now :)